platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -88,34 +86,14 @@ protected Object doExecute() {
 
                 outputWarningMessage(String.format(FAILED_EXPORT_MESSAGE, exportDirectory));
             }
-        } catch (Exception e) {
```

Comment:
The service call is made inside the lamba (line 77), which is wrapped inside the `security.runWithSubjectOrElevate()` method. If the service throws any kind of exception, `runWithSubjectOrElevate()` will wrap it inside an `InvocationTargetException`. So if the service throws a `MigrationException`, it will be wrapped and caught by the `catch (InvocationTargetException)` block and the `e.getCause().getMessage()` will print the right message.

Hope that makes sense.